SPECS/msft-golang/msft-golang.spec

```diff
@@ -12,15 +13,16 @@
 %define __find_requires %{nil}
 Summary:        Go
 Name:           msft-golang
-Version:        1.19.12
+Version:        1.20.11
```

Comment:
Don't we need to release-bump all packages depending on `golang`?

Reply:
We would for the `golang` package, but this is `msft-golang`. I don't see any packages that have a BR on it.

Reply:
Notes on this variant of go: https://github.com/microsoft/CBL-Mariner/pull/2964

Reply:
Ah, I missed the fact this is not the general `golang` package.